instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go

```diff
@@ -349,13 +350,22 @@ func UnaryServerInterceptor(opts ...Option) grpc.UnaryServerInterceptor {
 
 		messageReceived.Event(ctx, 1, req)
 
+		var statusCode grpc_codes.Code
+		defer func(t time.Time) {
+			elapsedTime := float64(time.Since(t)) / float64(time.Millisecond)
```

Comment:
Should this instrument be defined for `int64` instead of `float64` given there will always be a conversion of int to float here?

Reply:
I think `float64` may be better? Since `float64` has better precision than `int64`, and `http.server.duration` of  http instrumentation library also made choice of using `float64` other than `int64` in issue #1537

Reply:
Shouldn't times/durations be represented as `int64` nanoseconds?

Reply:
I think we should use `float64` other than `int64`. 

Since java http instrumentation and golang http instrumentation both use float 
- java: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/c518bf8d7ef2ac6f409621561f6e2dcd09fbb510/instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpServerMetrics.java#L52
- golang: https://github.com/open-telemetry/opentelemetry-go-contrib/blob/773172b3d749ce3b18b6b4c691ccb2e341862fa0/instrumentation/net/http/otelhttp/handler.go#L111

And I also search for the duration metric semantic conventions, but found this is still in discussion 
- https://github.com/open-telemetry/opentelemetry-specification/issues/654
- https://github.com/open-telemetry/oteps/pull/129
- https://github.com/open-telemetry/opentelemetry-specification/pull/1142
- https://github.com/open-telemetry/opentelemetry-specification/pull/1177

Since the semantic is not finally defined, should we keep consistence with other existing implemetations?

@Aneurysm9 @MrAlias

Reply:
> Since the semantic is not finally defined, should we keep consistence with other existing implemetations?

I'm more inclined to say that we shouldn't add metrics at all until the conventions are better established.

Reply:
I already changed the metric from `SyncFloat64` to `SyncInt64`. @MrAlias